flopy/mf6/mfmodel.py

```diff
@@ -1151,6 +1152,76 @@ def remove_package(self, package_name):
             for child_package in child_package_list:
                 self._remove_package_from_dictionaries(child_package)
 
+    def update_package_filename(self, package, new_name):
+        """
+        Updates the filename for a package.  For internal flopy use only.
+
+        Parameters
+        ----------
+        package : MFPackage
+            Package object
+        new_name : str
+            New package name
+        """
+        try:
+            # get namefile package data
+            package_data = self.name_file.packages.get_data()
+        except MFDataException as mfde:
+            message = (
+                f"Error occurred while updating package names "
+                "from name file in model "
+                '"{self.name}".'
```

Comment:
this should be:
```
                "Error occurred while updating package names "
                f'from name file in model "{self.name}".'
```

Reply:
thanks!  looks like Codacy caught that too.